includes/core/incSendMail.asp

```diff
@@ -19,10 +19,13 @@
 
 <script language="python" runat="server">
 from cioc.core.email import send_email,DeliveryException
-def l_send_email(author, to, subject, message, ignore_block):
-	print 'send_email'
+def l_send_email(author, to, subject, message, ignore_block, domain_override=None):
+	args = {}
+	if domain_override:
+		args['domain_override'] = domain_override
```

Comment:
extra complicated here so that in-place upgrade can happen before recycle/reload of cioc.core.email